### ISSUE TITLE
Add option to render toolbar above image

### ISSUE THREAD

**Issue by BassT** [infeng/react-viewer#105]
First of all, thanks for this component.

Currently, the toolbar is always rendered below the image.

I'd like to suggest the idea to add an option to render the toolbar above the image.

**Comment by infeng** [comment]
Do youou mean the one below?
![image](https://user-images.githubusercontent.com/16483610/81928669-37563700-9618-11ea-9194-3bc05203b11a.png)

**Comment by BassT** [comment]
Yes. Looking at the source an props it seemed to me that's called the toolbar.
